CMakeLists.txt

```diff
@@ -457,7 +457,7 @@ endif()
 add_subdirectory(boards)
 add_subdirectory(ext)
 add_subdirectory(subsys)
-add_subdirectory_ifdef(CONFIG_KERNEL drivers)
+add_subdirectory_ifdef(CONFIG_DRIVERS drivers)
```

Comment:
Try to make this non-conditional, AFAICS it will work without the kconfig check.